.github/workflows/get-pass-secrets.yaml

```diff
@@ -0,0 +1,49 @@
+name: Get and pass secrets
```

Comment:
Bad name for file and workflow itself, basically you don't only create and pass secrets here, you perform whole CI/CD

Reply:
Fixed